### PR TITLE
Update ETC.tsv

### PR DESCRIPTION
**Pull request by seejy**
line # 14494
line #14533
line #14594
Punctuation from line #14494 until #14595

### CODE REVIEW

**Comment by vtange** [comment]
There's a good chance a lot of the lines you just added periods to don't need periods, as they appear in the middle of the screen as a brief notifier,

**Comment by seejy** [comment]
I see. Is there a way to know which lines are brief notifiers? Thank you! :)

File: ETC.tsv [hunk 1]
```diff
@@ -14573,24 +14573,24 @@ ETC_20150918_014574	{memo X}The devices have been reset.
...
 ETC_20150918_014589	{memo X}Power
-ETC_20150918_014590	You obtained a part
+ETC_20150918_014590	You obtained a part!
 ETC_20150918_014591	Guards!
 ETC_20150918_014592	Intruders!
-ETC_20150918_014593	{memo X}Letting him know the place where Dellus is
-ETC_20150918_014594	There aren't usable equipment
-ETC_20150918_014595	Some field monsters were involved in battles so you couldn't summon them
+ETC_20150918_014593	{memo X}Letting him know the place where Dellus is...
+ETC_20150918_014594	There aren't any usable equipment
```
Comment (terrapin47) [inline ETC.tsv:14592]: isn't any

**Comment by terrapin47** [comment]
Not by just looking at the file.  I wouldn't add any punctuation in ETC.  Typically changes have already been made to make text fit or look correct in-game.

**Comment by seejy** [comment]
Ok then. Shall unchange all the changes then. Thanks :)

**Pull request closed without merging**